cdap-ui/app/cdap/components/Footer/index.js

```diff
@@ -35,7 +35,7 @@ export default class Footer extends Component {
           <div className="row text-muted">
             <div className="text-uppercase">
               <p className="text-xs-center">
-                <span>Copyright &copy; {this.copyrightYear} Cask Data, Inc.</span>
```

Comment:
With this change we don't need the `copyrightYear` prop and variable anymore right?

Reply:
Good catch. Will make this change.